RFC-001-BAM.md

```diff
@@ -247,13 +243,14 @@ A header's key acts as its identifier. A header's value MUST encode the followin
 
 2. A value: The actual value that is associated with the header
 
-    A header's value MUST be a single token, e.g. no nested structure etc. This is to facilitate comparisons of header-values. If a value needs to parameterized further, `parameters` should be used. Every header MUST have a value.
+    Every header MUST have a value. A header's value determines the parameters.
```

Comment:
```suggestion
    Every header MUST have a value. A header's value determines the (possibly empty) set of parameters.
```
Heh sorry suggesting changes to my previously suggested change.

Reply:
I think we should actually change this to not mention the parameters since they are explained in the next section anyway.